portal/server/utils/csv-utils.test.js

```diff
@@ -103,4 +109,38 @@ describe('csv-utils', () => {
       expect(mappedData).toEqual(expectedJsonData);
     });
   });
+
+  describe('extractCellValue', () => {
+    it('returns the cell value', async () => {
+      const cellValue = { value: 'GBP' };
+
+      const extractedValue = extractCellValue(cellValue);
+
+      expect(extractedValue).toEqual('GBP');
+    });
+
+    it('returns the cell value without new lines', async () => {
+      const cellValue = { value: 'GBP\na' };
```

Comment:
Would be nice to also test the case of multiple new line chars in different places.